esp-wifi/src/wifi/mod.rs

```diff
@@ -2619,6 +2591,14 @@ impl<'d> WifiController<'d> {
         Ok(())
     }
 
+    /// Configures modem power saving
+    ///
+    /// This is ignored, and set to `PowerSaveMode::Minimum` when `coex` is
```

Comment:
If it's ignored, can we remove the function when coex is enabled?

Reply:
Hmm I was thinking about this, but I was also thinking that we might remove the `coex` feature in the future as we can now dynamically spawn the coex task if we want to. I didn't run that by @bjoernQ yet though.

Reply:
I _think_ for coex ,both minimum and maximum might be suitable - but None is not allowed.

I'd say we will need to keep the `coex` feature - while we now can dynamically spawn and tear-down tasks there is much more conditional compilation going on when the `coex` feature is enabled. Even if things would "just work" I'm almost sure we would see a huge performance degradation if we would assume coex enabled in those places (but maybe it would just crash if `coex_init` isn't done)